src/pages/LoginPage/LoginPage.tsx

```diff
@@ -0,0 +1,28 @@
+import LoginPagePhoto from '../../assets/LoginPagePhoto.svg'
+import CircleLogo from '../../assets/stamp.svg'
+import './LoginPage.css'
+
+export default function LoginPage() {
+  return (
+    <div className='wrap'>
+      <div className='imgHolder'>
+        <img src={CircleLogo} className='imgLogo' />
+        <img src={LoginPagePhoto} alt='photo' className='img' />
```

Comment:
I suggest renaming this alt text to something more meaningful like `alt='login-screen-image'`